docs/DeveloperGuide.md

```diff
@@ -278,30 +278,30 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 | Priority | As a …​                  | I want to …​                                        | So that I can…​                                                    |
 |----------|--------------------------|-----------------------------------------------------|--------------------------------------------------------------------|
-| `* * *`  | business owner           | choose which company and point-of-contact to delete | save only relevant companies and point-of-contact to remove clutter |
+| `* * *`  | business owner           | choose which client and point-of-contact to delete | save only relevant companies and point-of-contact to remove clutter |
```

Comment:
Perhapse `companies` should be changed to clients? (This is recurring throughout the document)

Reply:
okay thanks for pointing out!